pkg/ibm_plugin/utils.go

```diff
@@ -94,6 +95,15 @@ func ZoneToRegion(zone string) (string, error) {
 	}
 }
 
+func GetZonesOfRegion(region string) []string {
```

Comment:
Can we use the cloud SDK's vpc service function instead of hardcoding:
`func (vpc *VpcV1) ListRegionZones(listRegionZonesOptions *ListRegionZonesOptions) (result *ZoneCollection, response *core.DetailedResponse, err error)`

Reply:
Sure.